src/main/resources/db/migration/V1_2__create_table_behandler_dialogmelding.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE BEHANDLER_DIALOGMELDING
+(
+    id            SERIAL PRIMARY KEY,
+    behandler_ref VARCHAR(50)  NOT NULL UNIQUE,
+    type          VARCHAR(30)  NOT NULL,
+    fnr           VARCHAR(11)  NOT NULL,
+    fornavn       VARCHAR(255) NOT NULL,
+    mellomnavn    VARCHAR(255),
+    etternavn     VARCHAR(255) NOT NULL,
+    partner_id    VARCHAR(50)  NOT NULL,
+    her_id        VARCHAR(50)  NOT NULL,
+    parent_her_id VARCHAR(50)  NOT NULL,
+    hpr_id        VARCHAR(50),
+    kontor        VARCHAR(255),
+    adresse       VARCHAR(255),
+    postnummer    VARCHAR(4),
+    poststed      VARCHAR(255),
+    orgnummer     VARCHAR(9),
+    created_at    timestamptz  NOT NULL,
+    updated_at    timestamptz  NOT NULL,
+    UNIQUE (partner_id, parent_her_id)
+);
+
+CREATE TABLE BEHANDLER_DIALOGMELDING_ARBEIDSTAKER
+(
+    id                         SERIAL PRIMARY KEY,
+    arbeidstaker_personident   VARCHAR(11) NOT NULL,
+    behandler_dialogmelding_id INTEGER REFERENCES BEHANDLER_DIALOGMELDING (id) ON DELETE CASCADE
+);
```

Comment:
Hvis vi skal ha dette som en egen tabell trenger vi i hvert fall en opprettet-timestamp:
`created_at    timestamptz  NOT NULL,` og helst en UUID som evt. kan brukes for esktern eksponering.